src/main/resources/db/migration/V42_8__Create_comment_table.sql

```diff
@@ -0,0 +1,6 @@
+create table if not exists comment
+(
+    id UUID PRIMARY KEY,
```

Comment:
No. It should be something like `id varchar primary key default uuid_generate_v4()`